src/main/java/com/rcv/StreamingCVRReader.java

```diff
@@ -198,7 +198,7 @@ private void endCVR() {
     }
     // provide some user feedback on the CVR count
     if (cvrList.size() % 50000 == 0) {
-      Logger.log(Level.INFO, String.format("Parsed %d cast vote records.", cvrList.size()));
+      Logger.log(Level.INFO, String.format("Parsed %d cast vote records...", cvrList.size()));
```

Comment:
... implies an action that's about to happen, so keep as period. Please apply to / revert other cases.